### PR TITLE
[Hot Fix] README figure wonk

### LINKED ISSUE #294: Fix README figure
There are some issues with the README pipeline figure

### PR DESCRIPTION
**Pull request by gwaybio**
# Description

closes #294

## What is the nature of your change?

- [x] Bug fix (fixes an issue).
- [ ] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +114,9 @@ Therefore, we have included some custom tools in `pycytominer/cyto_utils` that p
 - [Cell locations lookup table generation](#Creating-a-cell-locations-lookup-table)
 - [Generating gct files for Morpheus visualization](#Generating-a-GCT-file-for-morpheus)
 
+Note, [`pycytominer.cyto_utils.cells.SingleCells()`](pycytominer/cyto_utils/cells.py) contains code to interact with single cell SQLite files, which are output from CellProfiler.
+The single-cell processing speed of SQLite files depends on SQLite file size and amount of memory (RAM) available the computational environment machine.
```
Comment (d33bs) [inline README.md:118]: Suggesting some very optional changes here. I think SQLite extraction can be both memory and core intensive.
```suggestion
Processing capabilities for SQLite files depends on SQLite file size and your available computational resources (for ex. memory and cores).
```

File: README.md [hunk 2]
```diff
@@ -113,6 +114,9 @@ Therefore, we have included some custom tools in `pycytominer/cyto_utils` that p
 - [Cell locations lookup table generation](#Creating-a-cell-locations-lookup-table)
 - [Generating gct files for Morpheus visualization](#Generating-a-GCT-file-for-morpheus)
 
+Note, [`pycytominer.cyto_utils.cells.SingleCells()`](pycytominer/cyto_utils/cells.py) contains code to interact with single cell SQLite files, which are output from CellProfiler.
```
Comment (d33bs) [inline README.md:117]: Suggestion for keeping single-cell hyphenation consistent.
```suggestion
Note, [`pycytominer.cyto_utils.cells.SingleCells()`](pycytominer/cyto_utils/cells.py) contains code to interact with single-cell SQLite files, which are output from CellProfiler.
```

File: README.md [hunk 3]
```diff
@@ -10,8 +10,9 @@
 Pycytominer is a suite of common functions used to process high dimensional readouts from high-throughput cell experiments.
 The tool is most often used for processing data through the following pipeline:
 
-<img height="500" src="https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png?raw=true">
-[Click here for high resolution image](https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png)
+<img height="325" src="https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png?raw=true">
```
Comment (d33bs) [inline README.md:13]: Consider adding an `alt` attribute to this `img` which helps describe the image to increase web accessibility.

**Review by d33bs — APPROVED** [review]
Looks good! I left a few very minor suggestions, please let me know if you have any questions.

File: README.md (hunk 3 above)
Comment (gwaybio) [reply README.md]: good point, will add

**Pull request merged**